2_observations/src/data_munge_functions.R

```diff
@@ -37,7 +36,21 @@ filter_temp_data <- function(cross_ind, dat_ind, out_ind) {
 
 }
 
-munge_temp_dat <- function(sites_ind, dat_ind, out_ind) {
+mark_time_space_holdouts <- function(df, holdout_water_years, holdout_reach_ids){
+  time_holdout_days <- sapply(holdout_water_years, water_year_to_days) %>%
+    reduce(.f = c)
+  df %>% mutate(in_time_holdout = date %in% time_holdout_days,
+                in_space_holdout = seg_id_nat %in% holdout_reach_ids,
+                test = in_time_holdout | in_space_holdout)
```

Comment:
can you drop the `in_time_holdout` and `in_space_holdout` columns? We only need the test column.

Reply:
Hmm yeah. I was thinking that's how the narrative would be used, but you're right that it might be nice to know which are time/space holdouts. You can leave for now.